lib/git-user.js

```diff
@@ -9,7 +9,7 @@ module.exports = function () {
     email = exec('git config --get user.email')
   } catch (e) {}
 
-  name = name && name.toString().trim()
+  name = name && JSON.stringify(name.toString().trim()).slice(1, -1)
```

Comment:
My bad! I mean slash, not quote (The extra ones in \\"me\\"). :P
Anyway, it did clear things up.

BTW, I guess `JSON.stringify` is preferred over `String.replace` due to the slight performance hit?